vermouth/processors/merge_chains.py

```diff
@@ -84,7 +81,7 @@ def merge_chains(system, chains, all_chains):
 class MergeChains(Processor):
     name = 'MergeChains'
 
-    def __init__(self, chains=None, all_chains=False):
+    def __init__(self, chains=[], all_chains=False):
         self.chains = chains
         self.all_chains = all_chains
```

Comment:
It's generally a bad idea to give a mutable type as default argument, since it will be created only once and shared among all calls/instances. Here it's *probably* fine since `chains` is not modified anywhere, but still, something like this is preferred:
```py
def __init__(self, chains=None, all_chains=False):
    self.chains = chains or []
    self.all_chains = all_chains.
```

As an example of what can go wrong:
```py
def a(x=[]):
    x.append(1)
    print(x)

a()
a()
```

Reply:
Thanks!